networkx/readwrite/adjlist.py

```diff
@@ -302,12 +296,14 @@ def read_adjlist(path, comments="#", delimiter=None, create_using=None,
     """
     lines = (line.decode(encoding) for line in path)
     return parse_adjlist(lines,
-                         comments = comments,
-                         delimiter = delimiter,
-                         create_using = create_using,
-                         nodetype = nodetype)
+                         comments=comments,
+                         delimiter=delimiter,
+                         create_using=create_using,
+                         nodetype=nodetype)
 
 # fixture for nose tests
+
+
 def teardown_module(module):
     import os
     for fname in ['test.adjlist', 'test.adjlist.gz']:
```

Comment:
Can anyone explain what does `teardown_module` do? I guess its for tests, and it deletes the files that's created, but does this have to be here?